tests/gdimagecopymerge/Makemodule.am

```diff
@@ -0,0 +1,7 @@
+if HAVE_LIBPNG
+libgd_test_programs += \
+	gdimagecopymerge/gdimagecopymerge
+endif
+
+EXTRA_DIST += \
```

Comment:
shouldn't this be including the new cmake file ?

Reply:
sorry... missed it